lib/presentation/ui/pages/login/login_view.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_base_rootstrap/presentation/ui/pages/login/widget/input_widget.dart';
+
+class LoginView extends StatelessWidget {
+  const LoginView({Key? key}) : super(key: key);
+
+  static const double _headerHeightPercentage = 0.3;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      color: Colors.white,
+      child: Column(
+        children: [
+          Container(
+            width: MediaQuery.of(context).size.width,
+            height:
+                MediaQuery.of(context).size.height * _headerHeightPercentage,
+            decoration: const BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage('assets/images/header.png'),
+                fit: BoxFit.fill,
+              ),
+            ),
+            child: const Center(
+              child: Text(
+                'TARGET MVD',
```

Comment:
don't use plain strings, move this to the intl file and use S.of(context) to get the value.
same for others.